apps/drain-safe/src/Container.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+export default styled.form`
```

Comment:
Do you know if this would be a violation of this rule? https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-anonymous-default-export.md

It can lead to some react-specific downsides: https://twitter.com/dan_abramov/status/1255229440860262400

I think in this specific case it would be shown as styled.form in the devtools, but not sure about the others 🤔